os/kernel/binary_manager/binary_manager.h

```diff
@@ -60,17 +60,11 @@
 #define BIN_TYPE_BIN               0                          /* 'bin' type for kernel binary */
 #define BIN_TYPE_ELF               1                          /* 'elf' type for user binary */
 
-/* Binary information configuration */
-#define PARTS_PER_BIN              2                          /* The number of partitions per binary */
-#define BIN_VER_MAX                16                         /* The maximum length of binary version */
-#define KERNEL_VER_MAX             8                          /* The maximum length of kernel version */
+#define FILES_PER_BIN              2                          /* The number of files per binary */
```

Comment:
This is not used anywhere. Please check this.

Reply:
It is used in binary_manager_load function.